src/components/Header/Navigations.tsx

```diff
@@ -42,20 +42,20 @@ const Navigations = ({ direction }: { direction?: string }) => {
           whileHover={{ scale: 1.1 }}
           className="md:text-sm lg:text-md text-base text-textColor cursor-pointer hover:text-headingColor duration-100 transition-all ease-in-out"
         >
-          <Link to={'/services'}>Services</Link>
+          {/* <Link to={'/services'}>Services</Link>
```

Comment:
Let's just remove the code instead of commenting it out. We can look at the commit history to recover the code if we need it in the future.